### PR TITLE
New package: python-pcapfile-0.12.0

### PR DESCRIPTION
**Pull request by cipr3s**

### CODE REVIEW

**Comment by cipr3s** [comment]
The upstream's tarball doesn't include any ISC license file.

File: srcpkgs/python-pcapfile/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=python-pcapfile
+version=0.12.0
+revision=1
+wrksrc="pypcapfile-${version}"
+build_style=python2-module
+pycompile_module="pcapfile"
+hostmakedepends="python-setuptools"
+short_desc="A Python library for handling libpcap savefiles"
+maintainer="cipr3s <cipr3s@gmx.com>"
+license="ISC"
```
Comment (pullmoll) [inline srcpkgs/python-pcapfile/template]: The ISC license is not installed by `base-files`, which means you need to add a post_install() hook:
```
post_install() {
        vlicense LICENSE
}
```

**Comment by pullmoll** [comment]
Another `noarch=yes` case I assume.

**Comment by pullmoll** [comment]
> The upstream's tarball doesn't include any ISC license file.
You can perhaps `$XBPS_FETCH_CMD https://github.com/kisom/pypcapfile/blob/master/LICENSE` then.

**Comment by cipr3s** [comment]
Updated. Sorry about forgetting "noarch" now i know it's the first thing to add for most python modules..

File: srcpkgs/python-pcapfile/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python-pcapfile'
+pkgname=python-pcapfile
```
Comment (asergi) [inline srcpkgs/python-pcapfile/template]: Rename the package to `python-pypcapfile`.

File: srcpkgs/python-pcapfile/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+version=0.12.0
+revision=1
+wrksrc="pypcapfile-${version}"
+noarch=yes
+build_style=python2-module
+pycompile_module="pcapfile"
+hostmakedepends="python-setuptools"
```
Comment (asergi) [inline srcpkgs/python-pcapfile/template]: `depends="python"`

File: srcpkgs/python-pcapfile/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="A Python library for handling libpcap savefiles"
+maintainer="cipr3s <cipr3s@gmx.com>"
+license="ISC"
+homepage="http://kisom.github.com/pypcapfile"
+distfiles="${PYPI_SITE}/p/pypcapfile/pypcapfile-${version}.tar.gz"
+checksum="3328e2136980468b0906bbe05a5a6e4e506a6f1a8451cf80393814c31e6f3b7f"
+
+post_install() {
+	$XBPS_FETCH_CMD https://github.com/kisom/pypcapfile/blob/master/LICENSE
```
Comment (asergi) [inline srcpkgs/python-pcapfile/template]: `$XBPS_FETCH_CMD https://raw.githubusercontent.com/kisom/pypcapfile/master/LICENSE`

**Comment by cipr3s** [comment]
Updated.

**Comment by Vaelatern** [comment]
Any more comments?

**Comment by pullmoll** [comment]
Looks good to me, but then I'm not the python type.

**Pull request closed without merging**